pixiecore/cli/quickcmd.go

```diff
@@ -299,7 +299,7 @@ func netbootRecipe(parent *cobra.Command) {
 		Long: `https://network.xyz allows to boot multiple operating
 	systems and useful system utilities.`,
 		Run: func(cmd *cobra.Command, args []string) {
-			kernel := "https://boot.netboot.xyz/ipxe/netboot.xyz.lkrn"
+			kernel := "https://boot.netboot.xyz/ipxe/netboot.xyz-packet-arm64.efi"
```

Comment:
This should switch based on architecture I think

Reply:
Yup, good catch.

For now, I've just switched it back and left it alone since netboot.xyz's arm64 support didn't work for my rpi4, so I can't really test it well.
It gave a menu, but it couldn't get network settings (so either missing an ethernet driver, or somehow specific to packet.net's setup).
Seems easiest to just remove it for now.